webdriver-extension/index.bs

```diff
@@ -0,0 +1,546 @@
+  the <a>local end</a> would POST to `/session/23/sensor/` with the body:
+  <pre class="lang-json">
+  {
+    "mockSensorType": "ambient-light",
+    "maxSamplingFrequency": 60,
+    "minSamplingFrequency": 5
+  }
+  </pre>
+  Be aware that only one <a>mock sensor</a> of a given <a>mock sensor type</a> can be created in <a>current browsing context</a>,
+  otherwise a <a>WebDriver error</a> with <a>WebDriver error code</a> [=mock sensor already created=] will be thrown.
+  
+</div>
+
+### Get mock sensor ### {#get-mock-sensor-command}
+
+<table>
+  <tbody>
+    <tr>
+      <th>HTTP Method</th>
+      <th><a lt="extension command uri template">URI Template</a></th>
+    </tr>
+    <tr>
+      <td>GET</td>
+      <td>/session/{session id}/sensor/{type}</td>
+    </tr>
+  </tbody>
+</table>
+
+The <dfn>get mock sensor</dfn> <a>extension command</a> simulates user
+getting information of a given type of <a>mock sensor</a>.
+
+The <a>remote end steps</a> are:
+1. Let |type| be a <a>url variable</a>, <a>converted to an IDL value</a> of type {{MockSensorType}}.
+   If this throws an exception, return a <a>WebDriver error</a> with <a>WebDriver error code</a>
+   <a>invalid argument</a>.
+2. If |type| does not match a [=mock sensor type=] amongst all associated [=mock sensors=] of the
+   <a>current browsing context</a>, return a <a>WebDriver error</a> with <a>WebDriver error code</a> [=no such mock sensor=].
+3. Return <a>success</a> with the <a>serialized mock sensor</a> as data.
+
+### Update mock sensor reading ### {#update-mock-sensor-reading-command}
+
+<table>
+  <tbody>
+    <tr>
+      <th>HTTP Method</th>
+      <th><a lt="extension command uri template">URI Template</a></th>
+    </tr>
+    <tr>
+      <td>POST</td>
+      <td>/session/{session id}/sensor/{type}</td>
+    </tr>
+  </tbody>
+</table>
+
+The <dfn>update mock sensor reading</dfn> <a>extension command</a> simulates user
+updating of a given type of <a>mock sensor</a>'s [=mock sensor reading|reading=].
+
+The <a>remote end steps</a> are:
+1. Let |type| be a <a>url variable</a>, <a>converted to an IDL value</a> of type {{MockSensorType}}.
+   If this throws an exception, return a <a>WebDriver error</a> with <a>WebDriver error code</a>
+   <a>invalid argument</a>.
+2. If |type| does not match a [=mock sensor type=] amongst all associated [=mock sensors=] of the
+   <a>current browsing context</a>, return a <a>WebDriver error</a> with <a>WebDriver error code</a> [=no such mock sensor=].
+3. Let |reading| be the |reading| argument, <a>converted to an IDL value</a> of the |type|'s
+   associated {{MockSensorReadingValues}}. If this throws an exception, return a
+   <a>WebDriver error</a> with <a>WebDriver error code</a> <a>invalid argument</a>.
+4. [=map/For each=] |key| → <var ignore>value</var> of |reading|.
+  1.  [=map/Set=] [=default mock sensor reading=][|key|] to the corresponding value of |reading|.
```

Comment:
Here again, we're referring to the concept of a "default mock sensor reading" when what's needed is the actual mock sensor reading that is being updated.